poc/idpf.sage

```diff
@@ -11,9 +11,13 @@ class Idpf:
     # Number of keys generated by the IPDF-key generation algorithm.
     SHARES: Unsigned = None
 
-    # Bit length of valid input values (i.e., values of `alpha`).
+    # Bit length of valid input values (i.e., the length of `alpha` in bits).
     BITS: Unsigned = None
 
+    # The length of each output vector (i.e., the length of `beta_leaf` and each
+    # element of `beta_inner`).
+    OUT_LEN: Unsigned = None
```

Comment:
Doesn't the output length depend on the number of prefixes? Or is this to support tuples as values?

Reply:
This is to support tuples as values. I guess the comments need to be clearer ...

Reply:
Maybe call it `VALUE_LEN` then, since it's the length of one output value, not the entire output?

Reply:
Oh that's better.

Reply:
Done.